javascript/Cpp/pitch.sed

```diff
@@ -0,0 +1,41 @@
+#
+# https://github.com/antlr/grammars-v4/blob/master/javascript/Cpp/README.md
+#
+# sed -f pitch.sed JavaScriptParser.g4 > JavaScriptParser-CppRuntime.g4
+# sed -f pitch.sed JavaScriptLexer.g4  > JavaScriptLexer-CppRuntime.g4
+#
+# Replease 'this.' TO 'this->'; 
+#
+s/this\./this->/g
+
+#
+# Add the @header{} section after the options {} section.
+#
+/options {/ {
+  h
+}
+
+/}/ {
+  p
+  H
+  g
+  /options {.*}/ {
+    a 
+    a // Auto generated by 'pitch.sed' script
+    a @header {
+    a // TODO: add some #include
+    a #include \"JavaScriptBaseLexer.h\"
+    a #include \"JavaScriptBaseParser.h\"
+    a }
+  }
+}
+
+/}/ {
+  z
+  h
+  d
+}
+
+/.*/ {
+  H
+}
```

Comment:
Is it a universal script for targeting .g4 files to C++ runtime? Can we place this file to the root directory and add info to README how to use it? As I understand the similar info is described in https://github.com/antlr/grammars-v4/tree/master/javascript/Cpp

@mike-lischke what do you think about it?

Reply:
> Is it a universal script for targeting .g4 files to C++ runtime?

Seems not yet, as there's hard coded #include, but maybe there's a chance to make it universal?

Reply:
Looks like this is some local stuff for the Javascript grammar with a C++ target. Nothing that should go into a global scope.

Reply:
Ok, I'm leaving it as is until better times.